src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(MAXIMUM_NUMBER);
```

Comment:
In this place, you store the random number.
But not store the random color and invoke the method right in Ball constructor.
Consider to do all in one style.